src/Uno.UWP/System/DispatcherQueueTimer.cs

```diff
@@ -0,0 +1,128 @@
+using System;
+using System.Threading;
+using Windows.Foundation;
+
+namespace Windows.System
+{
+	public partial class DispatcherQueueTimer 
```

Comment:
This class should have a destructor calling the `.Stop()` method.
+ partial declarations missing.

Reply:
Good point for the finalizer, added !

`partial` are irrelevant here, they are just causing silent errors.